mst/backup-types.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Types of backups in Managed Service for TimescaleDB
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
+product: cloud
+keywords: [backup]
+tags: [backup, binary backup, logical backup]
+---
+
+# Backups in Managed Service for TimescaleDB
+Timescale Cloud databases are automatically backed up, with full backups daily,
+and write-ahead log (WAL) recorded continuously. All backups are
+[encrypted][avien-encrypt]. Timescale Cloud uses [`pghoard`][pghoard], a PostgreSQL® backup
+daemon and restore tool, to store backup data in cloud object stores. The number of
+backups stored and the retention time of the backup depends on the service plan.
+
+<highlight type="important"> The size of logical backups, and the size of the
+Timescale Cloud backup that appears on the Timescale Cloud web console differs, in
+some cases significantly. Backup sizes that appear in the Timescale Cloud web
+console are for daily backups, before encryption and compression. To view the
+size of each database, including space consumed by indexes, you can use the `\l+` command at the psql prompt. </highlight> 
+
+The two types of backups are binary backups and logical backups. Full backups
```

Comment:
Can we move this section under the next heading, above the table?

Reply:
done!!